pkg/storaged/filesystem/mounting-dialog.jsx

```diff
@@ -57,10 +58,10 @@ export const mount_options = (opt_ro, extra_options, is_filesystem) => {
                       });
 };
 
-export const at_boot_input = (at_boot, is_filesystem) => {
+export const at_boot_input = (at_boot, is_visible) => {
     return SelectOne("at_boot", _("At boot"),
                      {
-                         visible: is_filesystem,
+                         visible: vals => !client.in_anaconda_mode() && (!is_visible || is_visible(vals)),
```

Comment:
I dont get why this changed here. if the && condition first leg is false (in anaconda mode) the latter will not be executed, so there is not need to change the existing code from what I see. So visible: vals => !client.in_anaconda_mode() && is_filesystem(vals) would do, right? 

No need to change that, just want to understand/review properly.

Reply:
> So visible: vals => !client.in_anaconda_mode() && is_filesystem(vals) would do, right?

`is_filesystem` can be undefined, so we have check for that case as well.

I have renamed `is_filesystem` to `is_visible`, because that seemed to be the more appropriate name, *shrug*